examples/server/lwm2mserver.c

```diff
@@ -394,20 +481,41 @@ static void prv_do_write_client(char *buffer, lwm2m_context_t *lwm2mH, bool part
     fprintf(stdout, "Syntax error !");
 }
 
+/**
+ * @brief Writes data to the LwM2M client.
```

Comment:
Since this is a wrapper function for `prv_do_write_client`, it should be documented as such.